### PR TITLE
hunspellDict.fr-any: Link `fr-moderne.dic` to `fr_FR.dic`

### PR DESCRIPTION
**Pull request by Ma27**
###### Motivation for this change

See #46940 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: nixos/doc/manual/release-notes/rl-1903.xml [hunk 1]
```diff
@@ -51,7 +51,10 @@
 
   <itemizedlist>
    <listitem>
-    <para />
+    <para>
+      The <literal>hunspellDicts.fr-any</literal> dictionary now ships with <literal>fr_FR.{aff,dic}</literal>
+      which is linked to <literal>fr-toutesvariantes.{aff,dic}</literal>.
```
[the remark this answers is not in the document]
Comment (Ma27) [reply nixos/doc/manual/release-notes/rl-1903.xml]: good catch, I forgot to update the release notes,thanks! :+1:

**Comment by Ma27** [comment]
fixed and rebased onto master.

**Comment by Ma27** [comment]
Ping @symphorien @jtojnar

**Comment by worldofpeace** [comment]
@Ma27 I don't see any reasons here for this not to be merged. 

19.03 has past now, can we have the release note in 19.09?

**Comment by Ma27** [comment]
@worldofpeace fixed

**Comment by worldofpeace** [comment]
Thanks for fixing this @Ma27 :sparkles: Especially so because you noted that you don't speak french.

**Pull request merged**